site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -128,11 +128,28 @@ public function uploadGradeable() {
         if (array_key_exists('ta_grading', $_POST)) {
             $values['ta_grading'] = $_POST['ta_grading'];
             if (array_key_exists('grade_inquiries', $_POST)) {
-                $values['grade_inquiry_allowed'] = $_POST['grade_inquiries'];
+                $values['grade_inquiry_allowed'] = $_POST['grade_inquiries'] ?? 'false';
```

Comment:
Is there a reason why the boolean value here (and elsewhere in this file) is a string?

Reply:
I believe this is an instance of getting values passed from html elements to php code, and never being converted to boolean. 

at first I tried using booleans, but it would fail later on in the process.